documentation/docs/07-hooks.md

```diff
@@ -2,11 +2,47 @@
 title: Hooks
 ---
 
-An optional `src/hooks.js` (or `src/hooks.ts`, or `src/hooks/index.js`) file exports three functions, all optional, that run on the server — `handle`, `handleError` and `handleFetch`.
+For both the client and the server, a range of app-wide hooks can be specified.
+
+### hooks.client.js
+
+An optional `src/hooks.client.js` (or `src/hooks.client.ts`, or `src/hooks.client/index.js`) file exports one functions, optional, that run on the client — `handleError`.
+
+> The location of this file can be [configured](/docs/configuration) as `config.kit.files.hooks.server`
+
+#### handleError
+
+If an error is thrown during loading or rendering, this function will be called with the `error`. This allows for two things:
+
+- you can send data to an error tracking service, for example
+- you can customise the formatting. The return value of the function becomes the value of `$page.error`. To make this type-safe, you can customize the expected shape through `App.PageError`. Note that `message` is always a required property, to guarantee a sensible fallback behavior.
```

Comment:
```suggestion
- you can customise the formatting. The return value of the function becomes the value of `$page.error`. To make this type-safe, you can customize the expected shape through `App.PageError`, which must include a `message: string` property.
```

Reply:
I liked the previous version better because it also explains why.